scripts/create_image.sh

```diff
@@ -15,7 +15,7 @@ source config/functions/functions
 ## Try to update Fenix
 check_update() {
 	cd $ROOT
-
+	[ "$FENIX_CHECK_UPDATE_NO" ] && return
```

Comment:
How to use `FENIX_CHECK_UPDATE_NO` ?

Reply:
FENIX_CHECK_UPDATE_NO=1 ./make

Reply:
for example ! if i make some local changes and try to build and check how its woks or not

Reply:
if we have another simple way plz write me ;)

Reply:
You can just execute `touch .ignore-update` to ignore update.

Reply:
ok ! this one what i need ;) tnx